disco_aws_automation/disco_autoscale.py

```diff
@@ -91,13 +91,13 @@ def _get_group_generator(self, group_names=None):
             if not next_token:
                 break
 
-    def _get_instance_generator(self, instance_ids=None, hostclass=None, group_name=None):
-        '''Yields autoscaled instances in current environment'''
+    def _get_instance_generator(self, hostclass=None, group_name=None):
+        """Yields autoscaled instances in current environment"""
         next_token = None
         while True:
             instances = throttled_call(
                 self.connection.get_all_autoscaling_instances,
-                instance_ids=instance_ids, next_token=next_token)
+                instance_ids=None, next_token=next_token)
```

Comment:
That is correct, we never pass the `instance_ids` parameter. I did not notice that boto default value for `instance_ids` is `None`; in that case we should remove the parameter altogether.